pxtblocks/blocklyloader.ts

```diff
@@ -684,6 +684,21 @@ namespace pxt.blocks {
     }
 
     function initContextMenu() {
+        // Translate the context menu for blocks.
+        let msg:any = Blockly.Msg;
+        msg.DUPLICATE_BLOCK = lf("Duplicate");
+        msg.REMOVE_COMMENT = lf("Remove Comment");
+        msg.ADD_COMMENT = lf("Add Comment");
+        msg.EXTERNAL_INPUTS = lf("External Inputs");
+        msg.INLINE_INPUTS = lf("Inline Inputs");
+        msg.EXPAND_BLOCK = lf("Expand Block");
+        msg.COLLAPSE_BLOCK = lf("Collapse Block");
+        msg.ENABLE_BLOCK = lf("Enable Block");
+        msg.DISABLE_BLOCK = lf("Disable Block");
+        msg.DELETE_BLOCK = lf("Delete Block");
+        msg.DELETE_X_BLOCKS = lf("Delete %1 Blocks");
+        msg.HELP = "Help";
```

Comment:
Context menu is provided for both workspace and blocks. The original `initContextMenu()` provides the context menu for workspace, and the translation is included in my first commit in this PR. However, the context menu for blocks is provided by "core/block_svg.js" in Blockly's repository.

The above lines are required to give translation for context menu for blocks.